src/utils/trust.involved.ts

```diff
@@ -100,25 +112,36 @@ export const getTrustInvolvedPage = (
   }
 };
 
-export const postTrustInvolvedPage = (
+export const postTrustInvolvedPage = async (
   req: Request,
   res: Response,
   next: NextFunction,
-  isUpdate: boolean
+  isUpdate: boolean,
+  isReview: boolean
 ) => {
   try {
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
     if (req.body.noMoreToAdd) {
-      return safeRedirect(res, getNextPage(isUpdate));
+      if (isReview) {
+        const appData = getApplicationData(req.session);
+        const trustInReview = getTrustInReview(appData);
+        const trustId = trustInReview?.trust_id ?? "";
+        moveTrustOutOfReview(appData, trustId);
```

Comment:
We don't need to get the `trustInReview` or `trustId` at all here; lines 128/129 can be removed, and we only pass `appData` to `moveTrustOutOfReview`

```suggestion
        moveTrustOutOfReview(appData);
```

Reply:
Fixed